From 8fc4f7621313d93d5ed494b1eee9ba6011745da2 Mon Sep 17 00:00:00 2001
From: Ivo De Decker <ivodd@debian.org>
Date: Fri, 21 Dec 2018 17:09:57 +0100
Subject: [PATCH 3/3] Example usage of PkgSourceVersion substvars

This commits show how ${dh:ProvidePkgSourceVersion} and
${dh:DebPkgSourceVersion:debhelper} can be used to create a dependency between
two packages from the same source version.

This commit is only an example. This strict dependency is probably not
necessary.
---
 debian/control | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/control b/debian/control
index 0ba40652..42a549b1 100644
--- a/debian/control
+++ b/debian/control
@@ -51,6 +51,7 @@ Provides: ${dh:CompatLevels},
           dh-sequence-dwz,
           dh-sequence-installinitramfs,
           dh-sequence-systemd,
+          ${dh:ProvidePkgSourceVersion},
 Multi-Arch: foreign
 Description: helper programs for debian/rules
  A collection of programs that can be used in a debian/rules file to
@@ -66,5 +67,6 @@ Architecture: all
 Multi-Arch: foreign
 Depends: debhelper (>= 9.20160709),
          ${misc:Depends},
+         ${dh:DepPkgSourceVersion:debhelper},
 Description: debhelper add-on to handle systemd unit files - transitional package
  This package is for transitional purposes and can be removed safely.
-- 
2.11.0

